content/snippets/components/index.md

```diff
@@ -7,15 +7,15 @@ order: 0
 navigationTitle: "Components"
 
 title: "CSS Components"
-description: "Stylify CSS Components - Buttons, Grids, Forms, Animations, Switches and a lot more. Copy&Paste without CSS framework."
+description: "Stylify CSS Components - Buttons, Grids, Forms, Animations, Switches and a lot more. Copy&Paste without a CSS framework."
 ---
 
 Stylify components are inspired by the [Material Design 3](https://m3.material.io/).
 
-Components proportions are taken from their [community figma file](https://www.figma.com/file/DJMWYdOn4HyZ9GQqrGZfXD/Material-3-Design-Kit-(Community)?node-id=47909%3A2).
+Components' proportions are taken from their [community figma file](https://www.figma.com/file/DJMWYdOn4HyZ9GQqrGZfXD/Material-3-Design-Kit-(Community)?node-id=47909%3A2).
```

Comment:
Isn't this comma unnecessary?

Reply:
Actually that was intentional, since `Components` is a plural and ends with an **S**, Components' should be used, not Components's or any other combination.

More info available here [How To Make Words That End In “S” Possessive](https://www.thesaurus.com/e/grammar/whats-the-rule-for-doing-a-possessive-after-the-word-s/) under *"Plural nouns ending in S"* section - if anyone is interested. 😀

Reply:
Ok, I have checked the guide, and it seems ok. Have never heard about that. Thanks!

Reply:
> We learn something new each day. 😀

Thanks for the merge, I am planning to use `Stilify` in one of my upcoming projects so if I find any issues that need to solved, I'll help for sure because I really like the concept behind `Stilify`. ✌🏼

Reply:
Thanks! I am glad to hear it. 

In case of any questions/bugs/ideas, create an issue/discussion or contact me on discord :+1: